src/Surfnet/ServiceProviderDashboard/Infrastructure/DashboardBundle/Validator/Constraints/AtLeastOneSelected.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/**
+ * Copyright 2024 SURFnet B.V.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+declare(strict_types=1);
+
+namespace Surfnet\ServiceProviderDashboard\Infrastructure\DashboardBundle\Validator\Constraints;
+
+use Attribute;
+use InvalidArgumentException;
+use Symfony\Component\Validator\Attribute\HasNamedArguments;
+use Symfony\Component\Validator\Constraint;
+
+#[Attribute]
+class AtLeastOneSelected extends Constraint
+{
+
+    /**
+     * @param string[] $fieldNames
+     */
+    #[HasNamedArguments]
+    public function __construct(
+        public array $fieldNames,
+        ?array $groups = null,
+        mixed $payload = null,
```

Comment:
Isn't it possible to omit `mixed` here? If you use mixed here then I would also expect a test for different types of variables.